src/BaseButton.lua

```diff
@@ -66,8 +76,7 @@ function BaseButton:render()
 		modifier = Enum.StudioStyleGuideModifier.Hover
 	end
 	return withTheme(function(theme)
-		-- extract and join props? (allows children, alignments, etc)
-		return Roact.createElement("TextButton", {
+		local scrubbedProps = joinDictionaries(self.props, propsToScrub, {
 			Size = self.props.Size,
```

Comment:
Looks like these lines (80-84) are redundant now that we are applying the scrubbed self.props directly.